WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/DiscountTypeBottomSheetListSelectorCommand.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import Yosemite
+
+/// `BottomSheetListSelectorCommand` for selecting a discount type for a coupon.
+///
+final class DiscountTypeBottomSheetListSelectorCommand: BottomSheetListSelectorCommand {
```

Comment:
nit: some unit tests might be helpful, like for the initial `selected` and selection behavior 🙂  (there are probably some tests for other `BottomSheetListSelectorCommand` subclasses)

Reply:
Makes sense! I added tests in 68f4bd030ce0474c794a7e2d1558c194622d1539 and 21cd505e3e3e2079884cf8f0a134199f40e17e4f.